src/intervals/arithmetic.jl

```diff
@@ -567,3 +567,15 @@ isinteger(a::Interval) = (a.lo == a.hi) && isinteger(a.lo)
 
 convert(::Type{Integer}, a::Interval) = isinteger(a) ?
         convert(Integer, a.lo) : throw(ArgumentError("Cannot convert $a to integer"))
+
+
+"""
+    mince(x::Interval, n)
+
+Splits `x` in `n` intervals of the same diameter, which are returned
+as a vector.
+"""
+function mince(x::Interval, n)
+    nodes = range(x.lo, stop = x.hi, length = n+1)
```

Comment:
You don't need  `stop` I think for Julia >= 1.1?

Reply:
Maybe not; I'll check it now, and if it is not needed I'll push a new commit with this correction.

Reply:
You were right; it is not needed. I just pushed a new commit which removes it.